core/src/main/scala/kafka/server/AlterIsrManager.scala

```diff
@@ -70,7 +70,8 @@ object AlterIsrManager {
     time: Time,
     metrics: Metrics,
     threadNamePrefix: Option[String],
-    brokerEpochSupplier: () => Long
+    brokerEpochSupplier: () => Long,
+    brokerId: Int
```

Comment:
why can't we use config.brokerId?  Is this to support automatic broker id assignment in the pre-KIP-500 code?  It would be good to add a note to the commit, if so.

Reply:
This supplies the broker epoch rather than the brokerId